### ISSUE TITLE
LSA Whisperer Artifact - Build Workflow - Authenticated Users Only

### ISSUE THREAD

**Issue by secdev02** [EvanMcBroom/lsa-whisperer#9]
## System information

It appears the Build Artifacts are only available to users who are authenticated to GitHub. 
This may be by design.

## Description

The Build Artifact release files are only available to authenticated accounts.

Example Authenticated :
![Screenshot 2024-08-25 at 12 26 20 PM](https://github.com/user-attachments/assets/d430cd3b-1676-41d1-a74c-318da6e15a0a)

Unauthenticated there is no download option:

<img width="950" alt="Screenshot 2024-08-25 at 12 26 55 PM" src="https://github.com/user-attachments/assets/74657531-a3de-48b6-b31c-ec8d6888cc31">

**Comment by EvanMcBroom** [comment]
I found that the requirement to be authenticated to download a workflow artifact is made by GitHub and I cannot change that. To allow unauthenticated users to download built copies of the tool I added a job to the build workflow which will keep the `latest` release on the releases page up to date with the artifacts from the build workflow. Unauthenticated users may download the tool from there and I updated the readme to direct people to use that location to download the project.

**Comment by EvanMcBroom** [comment]
I found that the requirement to be authenticated to download a workflow artifact is made by GitHub and I cannot change that. To allow unauthenticated users to download built copies of the tool I added a job to the build workflow which will keep the `latest` release on the releases page up to date with the artifacts from the build workflow. Unauthenticated users may download the tool from there and I updated the readme to direct people to use that location to download the project.
